raspberry/main.py

```diff
@@ -411,6 +413,6 @@ def read_ToF_run_kf_and_LQR():
 
             # TODO change this to print the dataframe (all rows and columns)
             if enable_printing_cache_to_screen[0]:
-                print(f"time: {time_cache} \naltitude: {tof.altitude_cache()} \nTref: {throttle_ref_cache} \nyaw: {yaw_cache} \npitch: {pitch_cache} \nroll:{roll_cache} \nacc: {accelerometer_cache_} \nalti_ref_mts{altitude_reference_cache_mts}")
+                print(f"time: {time_cache} \naltitude: {tof.get_altitude_cache()} \nTref: {throttle_ref_cache} \nyaw: {yaw_cache} \npitch: {pitch_cache} \nroll:{roll_cache} \nacc: {accelerometer_cache_} \nalti_ref_mts{altitude_reference_cache_mts}")
```

Comment:
I guess this print statement is not necessary except for debugging

Reply:
Think there is a lot not needed for cache saving but its dealt with in PR #202

Reply:
Should we merge this PR first to make PR #202 include changes in TOF sensor?